lib/saml/post_url_service.rb

```diff
@@ -41,7 +41,7 @@ def initialize(saml_settings, session: nil, user: nil, params: {}, loa3_context:
     def should_uplevel?
       # handles uplevel checks for ID.me & Login.gov 'two-trip' logins
       # verified_at is a Login.gov atttribute indicating the presence of IAL2 status
-      user.verified_at.present? ? (user.loa[:current] < LOA::THREE) : (user.loa[:current] < user.loa[:highest])
+      user.verified_at.present? && user.loa[:current] < LOA::THREE
```

Comment:
Can we change the comment here as well, to reflect that this is no longer relevant with ID.me?